src/views/Hearing/Sidebar.js

```diff
@@ -102,17 +104,29 @@ class Sidebar extends React.Component {
               <div className="sidebar-section contents">
                 <h4><FormattedMessage id="table-of-content"/></h4>
                 <ListGroup>
-                  <ListGroupItem href="#hearing">
+                  {!isQuestionView && <ListGroupItem className={currentlyViewed === '#hearing' && 'active'} href="#hearing">
```

Comment:
Here is architectural challenge as these two use different "container" or "view" -component.
This is basically new feature and would take quite a lot of refactoring / writing new code.